featuretools/entityset/entityset.py

```diff
@@ -1269,11 +1269,15 @@ def add_last_time_indexes(self, updated_dataframes=None):
                         lti_df = pd.Series([], dtype="object")
                     else:
                         if lti_is_spark:
-                            lti_df["last_time"] = ps.to_datetime(lti_df["last_time"])
-                            lti_df["last_time_old"] = ps.to_datetime(
-                                lti_df["last_time_old"],
-                            )
                             # TODO: Figure out a workaround for fillna and replace
+                            if lti_df["last_time_old"].dtype != "datetime64[ns]":
+                                lti_df["last_time_old"] = ps.to_datetime(
+                                    lti_df["last_time_old"],
+                                )
+                            if lti_df["last_time"].dtype != "datetime64[ns]":
+                                lti_df["last_time"] = ps.to_datetime(
```

Comment:
The pyarrow upgrade made it such that calling `ps.to_datetime` on a column that is already datetime resulted in errors. We don't have woodwork initialized on lti_df, so I just did a dtype check here.